explainaboard/builders/extractive_qa.py

```diff
@@ -1,22 +1,10 @@
-from dataclasses import dataclass, field, fields
-from typing import Any, ClassVar, Dict, List, Optional
-import os
-import sys
-sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
-from info import *
-import feature
 from typing import Iterable
-from info import SysOutputInfo
-from info import BucketPerformance
-from info import Performance
-from info import Table
+from explainaboard.info import SysOutputInfo, BucketPerformance, Performance, Table
```

Comment:
If also mentioning some details, I think this format would be better. lol
```python
from explainaboard.info import (
    SysOutputInfo, 
    BucketPerformance, 
    Performance, 
    Table,
)
```
It can reduce editing conflicts among users to some extent.